### PR TITLE
Change mentions of "Perl 6" to "Raku"

### LINKED ISSUE #554: Switch to calling Perl 6 "Raku" in the README
And maybe in other places in the repository.

### PR DESCRIPTION
**Pull request by QhpAptyj9hj0RQwMOw3ujuO3GDP3kYzkmXSh4gk**
Documentation rename/update for #554

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -68,22 +68,22 @@ <h2 id="getting-started">Getting started</h2>
 <h3 id="installation-with-zef">Installation (with zef)</h3>
 <p>If you're just planning to be an Alma end user, <code>zef</code> is the recommended way to install Alma:</p>
 <div class="sourceCode"><pre class="sourceCode sh"><code class="sourceCode bash"><span class="ex">zef</span> install alma</code></pre></div>
-<p>In order to get the <code>zef</code> installer, you first need <a href="https://perl6.org/downloads/">Rakudo Perl 6</a>. Instructions for how to install <code>zef</code> itself can be found in the <a href="https://github.com/ugexe/zef#installation"><code>zef</code> README</a>.</p>
+<p>In order to get the <code>zef</code> installer, you first need <a href="https://raku.org/downloads/">Rakudo Raku</a>. Instructions for how to install <code>zef</code> itself can be found in the <a href="https://github.com/ugexe/zef#installation"><code>zef</code> README</a>.</p>
```
Comment (masak) [inline docs/index.html:71]: I think the link text should just say "Rakudo", not "Rakudo Raku".

**Comment by masak** [comment]
I think the Travis CI build has failed because I'm out of Travis CI credits (after a recent change in their policy). I'll try to switch us over to GitHub actions before we merge this PR.

File: docs/index.html (hunk 1 above)
Comment (QhpAptyj9hj0RQwMOw3ujuO3GDP3kYzkmXSh4gk) [reply docs/index.html]: Sure masak, I've updated this.

**Comment by QhpAptyj9hj0RQwMOw3ujuO3GDP3kYzkmXSh4gk** [comment]
That makes a lot of sense. The build fail confused me a bit.

I've noticed another mention of "Perl 6," so I've also updated this. To tidy the whole thing up, I've merged the 4 commits into 1.

**Comment by masak** [comment]
LGTM :+1:

**Pull request merged**